CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## 0.4.3
+
+* Padding R and S components if smaller than 31 bytes in SECP256R1 keys [#52](https://github.com/hyperledger/besu-native/pull/52)
+
```

Comment:
Also should probably include a changelog for non-blocking PRNG https://github.com/hyperledger/besu-native/commit/6a73c7d2a437a7e503a57076cbc58e36b2b0fb9f